partcad-cli/src/partcad_cli/click/commands/list/packages.py

```
@@ -4,9 +4,13 @@
 
 
 @click.command(help="List imported packages")
+@click.option("-r", "--recursive", is_flag=True, help="Recursively process all imported packages")
+@click.argument("package", type=str, required=False, default=".")  # help="Package to retrieve the object from"
 @click.pass_obj
-def cli(ctx):
-    with Process("ListPackages", "this"):
+def cli(ctx, recursive, package):
```

Ruff (0.8.2)

Missing return type annotation for public function `cli`

Add return type annotation: `None`

(ANN201)